app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -60,46 +71,77 @@ export interface TalkDetailsProps {
    */
   bio: string
   /**
-   * The url of the talk
+   * The image of the speaker
    */
-  talkUrl?: string
+  imageUrl: string
   /**
-   * The time of the event
+   * The name of the speaker
    */
-  eventTime: string
+  fullName: string
+  /**
+   * The company where the speaker works
+   */
+  company: string
   /**
    * The social buttons of the speaker
    */
   socialButtons: Array<{ url: string; icon: IconProps["icon"] }>
   /**
    * If we have multiple speakers, we show them in a carousel
    */
-  isMultipleSpeakers: boolean
-  /**
-   * The carousel data
-   */
-  carouselData?: Array<DynamicCarouselItem>
+  isMultipleSpeakers: false
 }
 
+export type TalkDetailsProps = TalkDetailsMultipleSpeakersProps | TalkDetailsSingleSpeakerProps
+
 const talkBlob = require("../../../assets/images/talk-shape.png")
 const talkCurve = require("../../../assets/images/talk-curve.png")
 
 const SCREEN_WIDTH = Dimensions.get("screen").width
 
+const triviaShowProps = (schedule: ScheduledEvent): TalkDetailsProps => {
+  const talk = schedule.talk
+  const speakers = [schedule["speaker-2-2"], schedule["speaker-3"], schedule["speaker-3-2"]].filter(
+    Boolean,
+  )
+  return {
+    title: schedule["event-title"],
+    subtitle: `${formatDate(schedule["day-time"], "MMMM dd, h:mmaaa")} PT`,
+    description: stringOrPlaceholder(schedule["event-description"]),
+    talkUrl: talk?.["talk-url"],
+    eventTime: schedule["day-time"],
+    isMultipleSpeakers: true,
+    carouselData: speakers.map((speaker: Speaker) => ({
+      image: { uri: speaker?.["speaker-photo"].url },
+      imageStyle: { height: 320 },
+      subtitle: speaker?.name,
+      label: speaker?.company,
+      body: stringOrPlaceholder(speaker?.["speaker-bio"]),
+      socialButtons: [
+        { url: speaker?.twitter, icon: "twitter" },
+        { url: speaker?.github, icon: "github" },
+        { url: speaker?.externalURL || speaker?.website, icon: "link" },
+      ],
+    })),
+  }
+}
+
 const talkDetailsProps = (schedule: ScheduledEvent): TalkDetailsProps => {
   const talk = schedule.talk
 
+  if (schedule["event-title"] === WEBFLOW_MAP.triviaShow.title) return triviaShowProps(schedule)
```

Comment:
do we go contains here for the day of title change? hoping no other talk has trivia in it?

a fun roll of the dice for sure

Reply:
Is this title going to change? If so this wouldn't be the only place we'd have logic changes.

Reply:
yeah i'd a contains might bite us considering how sensitive this is right now